### PR TITLE
EOS-16256:Create IEM class for storing IEM alert message.

### PR DESCRIPTION
**Pull request by Prabh2301**

### CODE REVIEW

File: py-utils/src/utils/IEM_framework/IEM_framework.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class IEM:
```
Comment (sachinpunadikar) [inline py-utils/src/utils/IEM_framework/IEM_framework.py:1]: Need copyright text

File: py-utils/src/utils/IEM_framework/IEM_framework.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    severity=" "
+    source_id=" "
+    component_id=None
+    module_id=None
+    event_id=None
+    message=" "
+    params={ }
+
+    def __init__(self):
```
Comment (sachinpunadikar) [inline py-utils/src/utils/IEM_framework/IEM_framework.py:10]: Constructor should accept all the parameters

File: py-utils/src/utils/IEM_framework/IEM_framework.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    
+    def populate(self,severity:str, source_id:str, component_id, module_id, event_id, message:str,**params):
+        self.severity=severity
```
Comment (sachinpunadikar) [inline py-utils/src/utils/IEM_framework/IEM_framework.py:13]: Add validation code for each argument being passed

File: py-utils/src/utils/IEM_framework/IEM_framework.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+        self.source_id=source_id
+        self.component_id=component_id
+        self.module_id=module_id
+        self.event_id=event_id
+    
+
```
Comment (sachinpunadikar) [inline py-utils/src/utils/IEM_framework/IEM_framework.py:19]: Add print method

**Review by sachinpunadikar — CHANGES_REQUESTED** [review]
Work on the review comments.
Additionally you need to add test cases for every dev code

**Review by selvakumaar5496 — CHANGES_REQUESTED** [review]

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Comment by sachinpunadikar** [comment]
No more needed

**Pull request closed without merging**